content/concepts/networks-and-servers/the-clio-server.md

```diff
@@ -36,6 +36,13 @@ A Clio server stores validated ledger data such as transaction metadata, account
 
 When a Clio server receives an API request, it looks up data from these data stores. For requests that require data from the P2P network, the Clio server forwards the request to a P2P server, and then passes the response back to the client.
 
+Clio will **always** forward to `rippled` if any of the following is true:
+
+- `ledger_index` is set to `current` or `closed`.
+- `api_version` is set to `1` - Clio only supports API v2.
+- `accounts`, `queue` or `full` are set to `true` for the `ledger` API.
```

Comment:
Why?

Reply:
This one is still true - Clio does not support these flags so best we can do is forward.